app/presenters/image_upload_response_presenter.rb

```diff
@@ -89,4 +91,16 @@ def doc_type_supported?
   def failed_fingerprints
     @form_response.extra[:failed_image_fingerprints] || { front: [], back: [] }
   end
+
+  def show_selfie_failures
```

Comment:
There's nothing being `show`n. I think I'll change this to `include`.

Reply:
I ended up keeping this the way it was because the naming matches the existing method that does something similar.